kmip/core/messages/payloads/add_attribute.py

```diff
@@ -0,0 +1,270 @@
+        """
+        Write the data encoding the AddAttribute request payload to a
+        stream.
+
+        Args:
+            ostream (stream): A data stream in which to encode object data,
+                supporting a write method; usually a BytearrayStream object.
+        """
+        tstream = utils.BytearrayStream()
+
+        if self.uid:
+            uid = primitives.TextString(
+                value=self.uid, tag=enums.Tags.UNIQUE_IDENTIFIER)
+            uid.write(tstream)
+
+        if self.attribute:
+            self.attribute.write(tstream)
+
+        self.length = tstream.length()
+        super(AddAttributeRequestPayload, self).write(ostream)
+        ostream.write(tstream.buffer)
+
+    def validate(self):
+        """
+        Error check the attributes of the AddAttribute request payload.
+        """
+        if self.uid is not None:
+            if not isinstance(self.uid, six.string_types):
+                raise TypeError(
+                    "uid must be a string; "
+                    "expected (one of): {0}, observed: {1}".format(
+                        six.string_types, type(self.uid)))
+        if self.attribute is not None:
+            if not isinstance(self.attribute, objects.Attribute):
+                raise TypeError(
+                    "attribute must be a Attribute object, "
+                    "observed: {1}".format(type(self.attribute)))
+
+    def __repr__(self):
+        uid = "uid={0}".format(self.uid)
+        attribute = "attribute={0}".format(self.attribute)
+        return "AddAttributeRequestPayload({0},{1})".format(
+            uid,
+            attribute)
+
+    def __str__(self):
+        return str({'uid': self.uid, 'attribute': self.attribute})
+
+    def __eq__(self, other):
+        if isinstance(other, AddAttributeRequestPayload):
+            if self.uid != other.uid:
+                return False
+            elif self.attribute != other.attribute:
+                return False
+            else:
+                return True
+        else:
+            return NotImplemented
+
+    def __ne__(self, other):
+        if isinstance(other, AddAttributeRequestPayload):
+            return not self.__eq__(other)
+        else:
+            return NotImplemented
+
+
+class AddAttributeResponsePayload(primitives.Struct):
+    """
+    A response payload for the AddAttribute operation.
+
+    The payload will contain the ID of the managed object with which the
+    attributes are associated. It will also contain a list of attribute names
+    identifying the types of attributes associated with the aforementioned
+    managed object. See Section 4.13 of the KMIP 1.1 specification for more
+    information.
+
+    Attributes:
+        uid: The unique ID of the managed object with which the retrieved
+            attributes should be associated.
+        attribute_names: The list of attribute names of the attributes
+            associated with managed object identified by the uid above.
```

Comment:
`attribute_names` is used with the `GetAttributeList` response payload. This should just be a single `attribute`.